sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,293 @@
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AggregationType = "None" | "Average" | "Count" | "Minimum" | "Maximum" | "Total";
+
+// @public
+export interface BatchQuery {
+    azureResourceIds?: string[];
+    includeQueryStatistics?: boolean;
+    qualifiedNames?: string[];
+    query: string;
+    serverTimeoutInSeconds?: number;
+    timespan?: string;
+    workspace: string;
+    workspaceIds?: string[];
+    workspaces?: string[];
+}
+
+// @public
+export type ColumnDataType = string;
+
+// @public
+export const CommonDurations: {
+    readonly last7Days: "P7D";
+    readonly last3Days: "P3D";
+    readonly last2Days: "P2D";
+    readonly lastDay: "P1D";
+    readonly lastHour: "PT1H";
+    readonly last4Hours: "PT4H";
+    readonly last24Hours: "P1D";
+    readonly last48Hours: "P2D";
+    readonly last30Minutes: "PT30M";
+    readonly last5Minutes: "PT5M";
+};
+
+// @public
+export interface ErrorDetail {
+    additionalProperties?: any;
+    code: string;
+    message: string;
+    resources?: string[];
+    target?: string;
+    value?: string;
+}
+
+// @public
+export interface ErrorInfo {
+    additionalProperties?: any;
+    code: string;
+    details?: ErrorDetail[];
+    innererror?: ErrorInfo;
+    message: string;
+}
+
+// @public
+export interface GetMetricDefinitionsOptions extends OperationOptions {
+    metricNamespace?: string;
+}
+
+// @public
+export interface GetMetricDefinitionsResponse {
+    definitions: MetricDefinition[];
+}
+
+// @public
+export interface GetMetricNamespacesOptions {
+    startTime?: string;
+}
+
+// @public
+export interface GetMetricNamespacesResponse {
+    namespaces: MetricNamespace[];
+}
+
+// @public
+export interface LocalizableString {
+    localizedValue?: string;
+    value: string;
+}
+
+// @public
+export class LogsClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsClientOptions);
+    // (undocumented)
+    queryLogs(workspaceId: string, query: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
+    // (undocumented)
+    queryLogsBatch(batch: QueryLogsBatch, options?: QueryLogsBatchOptions): Promise<QueryLogsBatchResponse>;
+}
+
+// @public (undocumented)
+export interface LogsClientOptions extends PipelineOptions {
+    host?: string;
+}
+
+// @public
+export interface MetadataValue {
+    name?: LocalizableString;
+    value?: string;
+}
+
+// @public
+export interface Metric {
+    displayDescription: string;
+    errorCode?: string;
+    id: string;
+    name: LocalizableString;
+    timeseries: TimeSeriesElement[];
+    type: string;
+    unit: MetricUnit;
+}
+
+// @public
+export interface MetricAvailability {
+    retention?: string;
+    timeGrain?: string;
+}
+
+// @public
+export interface MetricColumn {
+    name?: string;
+    type?: ColumnDataType;
+}
+
+// @public
+export interface MetricDefinition {
+    category?: string;
+    dimensions?: LocalizableString_2[];
```

Comment:
can we avoid having a `_2` typename?

Reply:
We were just talking about this on the feature crew. I believe we're going to try to just type these as 'plain' string as the feature for localizing these strings isn't actually available.

CC: @srnagar

Reply:
Also captured here: https://github.com/Azure/azure-sdk-for-js/issues/15296